example/lib/main.dart

```diff
@@ -29,6 +30,26 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
+  String content = 'Logto SDK Demo Home Page';
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      _init();
+    });
+  }
+
+  void _init() async {
+    LogtoCore.fetchOidcConfig(
+            "https://logto.dev/oidc/.well-known/openid-configuration")
```

Comment:
nit: set this to a const, will be clear that this should be changed to real url.

Reply:
yes, will refactor this in the following PR.